galvani/BioLogic.py

```diff
@@ -244,6 +245,9 @@ def VMPdata_dtype_from_colIDs(colIDs):
                 unique_field_name = field_name
             type_list.append((unique_field_name, field_type))
         else:
+            print('ColIDs: ',
```

Comment:
I would rather not use `print` for this. We can either include it in the text of the exception or use the `logging` module

Reply:
I think printing out the entire list of columns is unnecessary. However, it would be handy to know the column name of the column previous to the unknown column. This way you would know the unknown column is the one after the column name in the error.

Something like:

```python
raise NotImplementedError("column ID {id} after column {prev} is not".format({id:colID, prev:type_list[-1][1]})
```
```

Reply:
That looks fine to me. We can also include more information if you think it would help debugging - just don't use the `print` function for that.

Reply:
output looks like this:

`NotImplementedError: Column ID 5 after column time/s is unknown`